.travis.yml

```diff
@@ -182,6 +182,21 @@ jobs:
         - ./tests/scripts/start-compose.sh ./tests/environment/docker-compose-all-amps-test.yml
         - ./tests/scripts/wait-for-alfresco-start.sh "http://localhost:8082/alfresco"
         - travis_wait 20 mvn install -f tests/tas-all-amps/pom.xml -Pall-tas-tests -Denvironment=default -DrunBugs=false
+    - name: "Pipeline All AMPs tests and push docker images"
+      jdk: openjdk11
+      if: fork = false AND (branch = master OR branch =~ /support\/.*/) AND type != pull_request
+      install:
+        # Build additional Docker image for repo and share for CI pipeline
+        - export TRAVIS_BUILD_NUMBER=${TRAVIS_BUILD_NUMBER}
+        - travis_retry travis_wait 20 mvn install -q -f tests/pipeline-all-amps/pom.xml -DskipTests
+        - docker login quay.io -u ${QUAY_USERNAME} -p ${QUAY_PASSWORD}
+        - travis_retry travis_wait 20 mvn clean install -f ./tests/pipeline-all-amps/pom.xml deploy -Prelease
```

Comment:
> Looking a lot better.
> 
> * Have you checked that the correct images are pushed to https://quay.io/repository/?
>   I don't see them, or rather I do do see the wrong one for share from 4 days ago.
>   It also appears never to be deleted. It should be deleted after a few days as it will be
>   of little value after that.
> * Have you checked that the image is NOT pushed to https://hub.docker.com/u/alfresco/ ?

The new build images are pushed to https://quay.io/repository/alfresco/alfresco-pipeline-all-amps-repo?tab=tags for repo and https://quay.io/repository/alfresco/alfresco-pipeline-all-amps-share?tab=info for share.

The images not pushed to Docker hub.